cmd/commands/init.go

```diff
@@ -19,11 +19,10 @@ var (
 	}
 )
 
-// setupFile aborts on error... or should we return it??
 // returns 1 iff it set a file, otherwise 0 (so we can add them)
 func setupFile(path, data string, perm os.FileMode) (int, error) {
 	_, err := os.Stat(path)
-	if !os.IsNotExist(err) {
+	if !os.IsNotExist(err) { //permission errors generated if use os.IsExist
```

Comment:
See this gist for a pretty informative description as to what is going on, looks like os.IsExist is not  appropriate here https://gist.github.com/mastef/05f46d3ab2f5ed6a6787#file-isexist_vs_isnotexist-go-L35-L43